src/app/games/kamisado/tests/KamisadoMove.spec.ts

```diff
@@ -13,13 +14,9 @@ describe('KamisadoMove', () => {
     });
     it('should have a bijective encoder', () => {
         const rules: KamisadoRules = KamisadoRules.get();
-        const minimax: KamisadoMinimax = new KamisadoMinimax(rules, 'KamisadoMinimax');
-        const node: KamisadoNode = rules.getInitialNode();
-        const moves: KamisadoMove[] = minimax.getListMoves(node);
-        moves.push(KamisadoMove.PASS);
-        for (const move of moves) {
-            EncoderTestUtils.expectToBeBijective(KamisadoMove.encoder, move);
-        }
+        const moveGenerator: KamisadoMoveGenerator = new KamisadoMoveGenerator();
+        MoveTestUtils.testFirstTurnMovesBijectivity(rules, moveGenerator, KamisadoMove.encoder);
+        EncoderTestUtils.expectToBeBijective(KamisadoMove.encoder, KamisadoMove.PASS);
```

Comment:
pass ok, first turn ok, what about second turn ?

Reply:
ce n'était pas présent dans le test précédent si je ne m'abuse ! Et kamisado t'as du move coord to coord + pass et c'est tout. Mais le MoveTestUtils pourrait générer les mouvements premier tour + deuxième tour en soit, qu'en dis-tu ? À partir du plateau initial, il génère first move, en prends un au hasard et genère second move pour tester la bijectivité. Vu que c'est pas rare les jeux dont le first et second move sont différents.

Reply:
Premier et second différent tour différent ouais, j'en compte 4.
Mais ou le premier tour suffit potentiellement pas,  ouais ça monte pas mal :D !

Reply:
on tickette cela ?

Reply:
écrit la description du ticket içi et moi je crée le ticket !

Reply:
[W] [easy] MoveTestUtils: test bijectivity on multiple turns instead of first turn only (refactor)

Reply:
cependant y a que 4 jeux ou le deuxième tour apporte déjà de nouvelles infos, ce serait plutôt vérifier partout qu'on a tout les types de moves testés.
Tu veux qu'on teste deux tours de bijectivités pour 4/35 jeux ?

Reply:
À discuter dans le ticket approprié je dirais, sinon on aura perdu la trace de la discussion !

Reply:
bha vu qu'il s'agit de déterminer si créer le ticket est utile, non, faut le décider avant d'encombrer le backlog